### DISCUSSION TITLE
DPS and Engine

### DISCUSSION THREAD

**Discussion by Leproide** [jilleb/MQB-FPA#16] [Q&A] [answered]
Hi!
I see you have reversed the dps system and i have a question:
The DPS profile change the motor map or change only suspension, acc, light etc?
I read on some forums that the DPS changes the engine map, but I doubt that the ECU has multiple maps
Tnx, have a nice day :)

**Comment by jilleb** [comment, marked as answer]
I can confirm that the engine has multiple maps, and the driving profiles are able to switch them.

**Reply by Leproide** [reply]
Wow! O.o
VW service tell to me the dps dont change maps... The usual answer they give when they don't know what to say.

U think is possible to switch the map without the DPS button and SWAP code?
I have a composition media
